templates/typescript/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
```

Comment:
Does this have a lot of failures if turn on strict?

Reply:
It sure did! I fixed 'em and pushed a new commit.